ckanext/geodatagov/templates/package/read.html

```diff
@@ -14,10 +14,15 @@ <h3>{{ _('Collection') }}</h3>
 {% set collection_package = h.get_collection_package(collection_package_id) %}
 <section class="module-content">
     <h3>{{ _('Collection') }}</h3>
-    <p>{{ _('This dataset is part of the following collection:') }}</p>
-    <ul class="dataset-list unstyled">
-        {% snippet "snippets/package_item.html", package=collection_package, truncate=75 %}
-    </ul>
+    {% if collection_package %}
+        <p>{{ _('This dataset is part of the following collection:') }}</p>
+        <ul class="dataset-list unstyled">
+            {% snippet "snippets/package_item.html", package=collection_package, truncate=75 %}
+        </ul>
+    {% else %}
```

Comment:
what about the case where a dataset is genuinely not part of a collection. will that also display this UI?

Reply:
No, the collection section will not be displayed if a dataset is not part of collection.